Makefile

```diff
@@ -1,12 +1,17 @@
 DEV_CONTAINER_TAG:=devcontainer
 
+# Load the environment variables. If this errors review the README.MD and create a .env file as instructed
+include .env
+export
+
 build: lint
 	go build .
 
 run: kind-create terraform-hack-init
-	@echo "==> Attempting to sourcing .env file"
-	if [ -f .env ]; then set -o allexport; . ./.env; set +o allexport; fi; \
-	go run . &
+	# Stop any previously running instance of the operator
+	$(shell [ -f run.pid ] && cat run.pid | xargs kill)
+	# Store the pid of the running instance in run.pid file
+	go run . & echo "$$$$" > run.pid
```

Comment:
@jjcollinge we talked about cleaning up the instance of the provider so `make integration-tests` can be re-run without things going badly. This does that by storing the PID of the instance in `run.pid` then cleaning up if the file exists before starting the provider.

Reply:
Seem like a reasonable solution?

Reply:
Cool looks good - maybe flock the pid file to ensure only one run. Maybe we revisit this if we orchestrate the integrations tests from Go eventually?

Reply:
Sorry, totally forgot we talked about that - yeah that is a much better approach. Deal lets keep this for now and I'll raise an issue to track moving to go orchestrating the operators start up/shutdown.